tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2163,6 +2164,11 @@ describe('scalar card', () => {
           start: {step: 0},
           end: {step: 5},
         });
+        // Workaround to align minMax state with minMaxSteps$
+        store.overrideSelector(getMetricsCardMinMax, {
+          minStep: 10,
+          maxStep: 30,
+        });
```

Comment:
Given that you've updated all the tests in this describe block with the same piece of code, this seems like a great use case for a `beforeEach`. Maybe you could even move the other redundant code there as well?

Reply:
There are still some tests don't need this (or even better not have this) 
the last one on L2246